builders/build-linux-rust/Dockerfile

```diff
@@ -55,4 +58,24 @@ RUN rustup component add clippy rustfmt
 RUN cargo search --limit 1
 
 # Install `sccache`
-RUN cargo install sccache --version 0.4.2
+RUN cargo install sccache --version "${SCCACHE_TAG}"
+
+# Install QDK framework for QNAP packages (QPKG)
+RUN set -eux; \
+    export DEBIAN_FRONTEND=noninteractive; \
+    git clone https://github.com/qnap-dev/QDK.git --branch "${QDK_TAG}"; \
+    cd QDK; \
+    ./InstallToUbuntu.sh install
+
+# Cleanup
+RUN set -eux; \
```

Comment:
I think cleanup needs to be the same `RUN` command that is causing the unneeded files to appear. Otherwise the resulting image will still contain all the trash in the intermediate layer.

Reply:
makes sense, changed

Reply:
I think that if the additional step is removing anything, it means that the intermediate steps didn't clean up after itself correctly. Can you inline the cleanup commands into appropriate `RUN` steps so that final cleanup step is not needed?

Reply:
`apt` clean up wasn't cleaning up anything, removed and added `docs`/`man` removal to each stage.